bd-logger/src/builder.rs

```diff
@@ -179,6 +180,34 @@ impl LoggerBuilder {
     let (crash_monitor_tx, crash_monitor_rx) = tokio::sync::oneshot::channel();
 
     let network_quality_provider = Arc::new(SimpleNetworkQualityProvider::default());
+
+    let feature_flags_manager = FeatureFlagsManager::new(
```

Comment:
Would it be hard to move all this into the `async {}` block and pass the write side of the FF to async_log_buffer in its start() call? I think that would let us do all these ops on the event loop and avoid inline IO and also make it easier to hook this all up to the runtime flags that can let us dynamically configure the parameters etc